draft-ietf-quic-transport.md

```diff
@@ -92,9 +92,12 @@ informative:
 
 --- abstract
 
-This document defines the core of the QUIC transport protocol.  Accompanying
-documents describe QUIC's loss detection and congestion control and the use of
-TLS for key negotiation.
+This document defines the core of the QUIC transport protocol. QUIC provides
+multiplexed streams for delivering ordered application data, flow control,
```

Comment:
maybe?
```suggestion
streams for structuring application data, flow control,
```

Reply:
I don't think "delivering ordered data" is quite right... at best this should be "ordered delivery of application data". But that is a tricky one, since there's no ordering across streams. Hence my suggestion of streams as something that an application uses for structuring data.

Also, I think "multiplexed" is an unnecessary detail for an overview.